sql/moz-fx-data-shared-prod/telemetry_derived/newtab_interactions_v1/query.sql

```diff
@@ -71,11 +101,21 @@ categorized_events AS (
     mozfun.map.get_key(event_details, "source") AS newtab_open_source,
     metrics.string.search_engine_private_engine_id AS default_search_engine,
     metrics.string.search_engine_default_engine_id AS default_private_search_engine,
+    metrics.boolean.pocket_is_signed_in,
+    metrics.boolean.pocket_enabled,
+    metrics.boolean.pocket_sponsored_stories_enabled,
+    metrics.boolean.topsites_enabled,
+    metrics.string.newtab_homepage_category,
+    metrics.string.newtab_newtab_category,
+    metrics.boolean.newtab_search_enabled,
     ping_info.experiments,
         -- ??? private_browsing_mode
-        -- Partially unique session attributes
+        -- Partially unique visit attributes
     mozfun.map.get_key(event_details, "telemetry_id") AS search_engine,
     mozfun.map.get_key(event_details, "search_access_point") AS search_access_point,
+    SAFE_CAST(
```

Comment:
Do we want to result in a null value here in case the casting fails or should we have some sort of default value/category in case that happens?

Reply:
Yes this should be null to indicate it's probably not in the pocket category and we want to preserve the `INT64` type